app/controllers/provider/admin/registry/policies_controller.rb

```diff
@@ -0,0 +1,40 @@
+#frozen_string_literal: true
+
+class Provider::Admin::Registry::PoliciesController < Provider::Admin::BaseController
+  activate_menu :account, :integrate, :policies
+
+  layout 'provider'
+
+  def index
+    @policies = Policies::PoliciesListService.call(current_account, builtin: false)
+  end
+
+  def create
+    policy = current_account.create(create_policy_params)
+    @policy = Policies::PoliciesListService.new
```

Comment:
If we're going to use the service, maybe rename the variable, as it does not return really a "policy". WDYT?

Reply:
you are right, we were trying to mimic APIcast behaviour... but probably doesn't make sense here